mongo/options/listdatabasesoptions.go

```diff
@@ -11,6 +11,10 @@ type ListDatabasesOptions struct {
 	// If true, only the Name field of the returned DatabaseSpecification objects will be populated. The default value
 	// is false.
 	NameOnly *bool
+
+	// If true, only the databases which the user is authorized to see will be returned. For more information about
+	// the behavior of this option, see https://docs.mongodb.com/manual/reference/privilege-actions/#find.
```

Comment:
nit: add a note about the default value

Reply:
I'd intentionally left this out because the behavior is different not only between major server versions but between minor versions in the 4.0.x cycle, but reading the linked docs page again, I think it's safe to document that the default value is `true`, so I've added a note.

@jyemin During your review, can you comment whether or not you think this is OK to document?